src/main/java/babysitter/BabySitter.java

```diff
@@ -33,50 +30,34 @@ boolean invalidUserEntry(int startTime, int endTime, int bedTime) {
 				if (invalids[i] == sc.nextInt()) {
 					invalidTracker = true;
 				}
-				invalidTracker = false;
```

Comment:
You don't want to set `invalidTracker` on every iteration, because if it gets set to `true` on one iteration and then `false` the next time, you'll end up returning `false` in a situation where there actually _was_ an invalid hour.